### PR TITLE
redesign for lessons page

### PR DESCRIPTION
**Pull request by ifiguer**
Almost, forgot... This pull request is for issue 509: https://github.com/TheOdinProject/theodinproject/issues/509

### CODE REVIEW

**Comment by KevinMulhern** [comment]
Hey @ifiguer, I checked this out locally and it looks nothing like the mock up? 😕 

There seems to be some parts missing. 

* The header at the top which has the course logo and course name
*  and the column at the left that contains the navigation for the lesson

Theres other things that should have been removed that are still there also.
* the breadcrumbs bar
* the adverts on the page

Whats going on?

**Comment by ifiguer** [comment]
:( It is very likely that I did something wrong when sending the pull request, because it definitely works on my machine. I'll look into it when I get out of work. Thanks for being patient with me. I'm new to all this.

**Comment by KevinMulhern** [comment]
Not problem, don't worry about it. You probably just forgot to commit your latest changes. It happens 😄

File: config/database.yml [hunk 1]
```diff
@@ -1,29 +0,0 @@
-# You may need to provide a username, in which case you'd use
```
Comment (arun1595) [inline config/database.yml]: Why is this file removed?

File: app/views/lessons/show.html.erb [hunk 2]
```diff
@@ -31,71 +12,45 @@
...
-            <div class="lnav-text">
-              <p>Next</p><p>Lesson</p>
-            </div>
-          <% end %>
-        </div>
+      <div class="card">
+        <h3>Solutions</h3>
       </div>
+
+      <h1 class="text-center">Got Questions?</h1>
```
Comment (arun1595) [inline app/views/lessons/show.html.erb]: Please use the `bottom_cta` partial. Have a look at [this](https://github.com/TheOdinProject/theodinproject/blob/redesign/app/views/static_pages/faq.html.erb#L10) for an example

File: config/database.yml (hunk 1 above)
Comment (ifiguer) [inline config/database.yml]: My bad, I added it back

**Comment by ifiguer** [comment]
BTW do I need to make another pull request or are you able to see here the latest commits? I updated it to use the bottom_cta partial and added the correct classes to the buttons (button--primary).

**Comment by KevinMulhern** [comment]
Hi @ifiguer I checked this out locally and its looks really good 😄 

The `database.yml` is still showing up in this PR. It shouldn't have to be changed for this. You can get it back to the way it was before and include it in this PR by doing this through git:
```
git checkout master config/database.yml
git commit --amend --no-edit
git push origin fix/redesign-lessons
```

Once thats done, this will be ready to merge.

Thank you for all your work on this one 😄

**Comment by KevinMulhern** [comment]
Thanks @ifiguer 😄 
LGTM 👍 👍 👍

**Comment by ifiguer** [comment]
Hi @KevinMulhern thanks for the very helpful instructions. I think we should be good now. One thing I should mention is that the buttons at the bottom (View Course, Next Lesson, etc.) aren't necessarily the best on smaller screens. I wasn't sure on how to best style them.

Also, the Mark Lesson Completed button works differently depending on whether the user is signed in (if they are not it redirects to the login page, and marks the lesson completed when the user is signed in) but I wasn't able to test if this worked properly. I'm hoping this was the intended behavior.

**Comment by KevinMulhern** [comment]
No worries about those things @ifiguer 😄 

The mobile design for the lesson page hasn't been looked into yet, we will cover that in the next redesign review though.

Yeah thats the intended behaviour for those buttons 😄 

We have to do some follow up work to standardise the lesson content for all the lessons, the work you've done here has put us in a great position to do that. Thanks again for all your help we really do appreciate it ❤️ 

I'm about to go on a merging spree with these open PR's and deploy them to staging :shipit: 
keep an eye on https://gitter.im/TheOdinProject/Contributing to see when thats done so you can see your work on our staging server 😄
